tests/safety_replay/test_safety_replay.py

```diff
@@ -19,7 +19,9 @@
   ReplayRoute("6fb4948a7ebe670e|2019-11-12--00-35-53.bz2", Panda.SAFETY_CHRYSLER),            # CHRYSLER.PACIFICA_2018_HYBRID
   ReplayRoute("791340bc01ed993d|2019-04-08--10-26-00.bz2", Panda.SAFETY_SUBARU,               # SUBARU.OUTBACK
               Panda.FLAG_SUBARU_GEN2),
-  ReplayRoute("76b83eb0245de90e|2020-03-05--19-16-05.bz2", Panda.SAFETY_VOLKSWAGEN_MQB),      # VOLKSWAGEN.GOLF (MK7)
+  ReplayRoute("76b83eb0245de90e|2020-03-05--19-16-05.bz2", Panda.SAFETY_VOLKSWAGEN_MQB),      # VOLKSWAGEN.GOLF (stock ACC)
+  ReplayRoute("3cfdec54aa035f3f|2022-10-13--14-58-58.bz2", Panda.SAFETY_VOLKSWAGEN_MQB,       # VOLKSWAGEN.GOLF (openpilot long)
```

Comment:
safety replay's failing on this route. are there cereal changes?
```
replaying 3cfdec54aa035f3f|2022-10-13--14-58-58.bz2 with safety mode 15, param 1, alternative experience 0
Traceback (most recent call last):
  File "capnp/lib/capnp.pyx", line 1064, in capnp.lib.capnp._DynamicStructReader._which
RuntimeError: Member was null.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./test_safety_replay.py", line 53, in <module>
    if not replay_drive(lr, mode, param, alt_exp):
  File "/tmp/openpilot/panda/tests/safety_replay/replay_drive.py", line 30, in replay_drive
    if msg.which() == 'sendcan':
  File "capnp/lib/capnp.pyx", line 1079, in capnp.lib.capnp._DynamicStructReader.which.__get__
  File "capnp/lib/capnp.pyx", line 1066, in capnp.lib.capnp._DynamicStructReader._which
capnp.lib.capnp.KjException: Attempted to call which on a non-union type
```

Reply:
Yes, this was driven on stock cereal but a slightly newer one than linked to the openpilot in Panda's Dockerfile.

Reply:
looks like bumping openpilot worked